deps/spidershim/test/value.cc

```diff
@@ -962,3 +963,49 @@ TEST(SpiderShim, Function) {
   EXPECT_TRUE(gen->IsFunction());
   EXPECT_FALSE(genObject->IsFunction());
 }
+
+template <typename TypedArray, int kElementSize>
+static Local<TypedArray> CreateAndCheck(Local<v8::ArrayBuffer> ab,
+                                        int byteOffset, int length) {
+  v8::Local<TypedArray> ta = TypedArray::New(ab, byteOffset, length);
+  EXPECT_EQ(byteOffset, static_cast<int>(ta->ByteOffset()));
+  EXPECT_EQ(length * kElementSize, static_cast<int>(ta->ByteLength()));
+  EXPECT_EQ(ab->GetContents().Data(), ta->Buffer()->GetContents().Data());
+  EXPECT_EQ(ta->ByteOffset(), TypedArray::Cast(*ta->ToObject())->ByteOffset());
+  return ta;
+}
+
+TEST(ArrayBuffer_NeuteringApi, Function) {
+  V8Engine engine;
```

Comment:
Can you please add a comment before this line explaining that this test has been adopted from V8?  There are some similar comments already in `value.cc`.